docs/src/notation.md

```diff
@@ -0,0 +1,41 @@
+# Notation overview
+
+Since manifolds include a reasonable amount of elements and functions, the following list tries to keep an overview of used notation throughout `Manifolds.jl`.
+The order is alphabetically by name.
+They might be used in a plain form within the code or when referring to that code.
+This is for example the case the calligraphic symbols.
+
+Within the documented functions the utf8 symbols are used whenever possible,
+as long as that renders still in $\TeX$ within this documentation.
+
+| Symbol | Description | Also used | Comment |
+|:--:|:--------------- |:--:|:-- |
+| $\times$ | Cartesian product of two manifolds | | see [`ProductManifold`](@ref) |
+| $^{\wedge}$ | (n-ary) Cartesian power of a manifold | | see [`PowerManifold`](@ref) |
+| $T^*_p \mathcal M$ | the cotangent space at $p$ | | |
+| $\xi$ | a cotangent vector from $T^*_p \mathcal M$ | $\xi_1, \xi_2,\ldots,\eta,\zeta$ | sometimes written with base point $\xi_p$. |
+| $n$ | dimension (of a manifold) | $n_1,n_2,\ldots,m, \operatorname{dim}(\mathcal M)$| for the real dimension sometimes also $\operatorname{dim}_{\mathbb R}(\mathcal M)$|
```

Comment:
In docstrings `\dim` is used instead of `\operatorname{dim}`. Is this due to a documentation rendering issue?

Reply:
It's mainly due to me having missed to replace those, after I noticed `\dim` is available. I replaced those two, too, now.